### PR TITLE
VendNumber 2.0

### PR DESCRIPTION
**Pull request by unkillbob**
# VendNumber 2.0

vend-number

Final miscellaneous touch ups before bumping 2.0:

- update contributors in package.json
- add a gif to the README
- tidy up `_executeOperation` to be more ESNexty

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,15 @@
...
-  "author": "Sarah Hui <sarah@vendhq.com>",
+  "contributors": [{
+    "name": "Sarah Hui",
+    "email": "sarah@vendhq.com",
+    "url": "https://github.com/sehsarah"
+  }, {
+    "name": "James Bunt",
+    "email": "james@vendhq.com",
+    "url": "https://github.com/unkillbob"
+  }],
```
Comment (chrisgreeff) [inline package.json]: Ah get me and @N1ck in on this!

**Comment by chrisgreeff** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (unkillbob) [inline package.json]: Sure I wasn't sure who's worked on it before, I only had the history of the copy Sarah made in weggie back in the day :+1:

**Comment by unkillbob** [comment]
@chrisgreeff https://github.com/vend/vend-number/commit/ce67a51f45124443928db2a80d7aa8bd1bb6db87

**Comment by chrisgreeff** [comment]
Looks great man! 2.0 this shiz

**Pull request merged**